docs/extensibility/resources.md

```diff
@@ -24,7 +25,8 @@ The `resource` section is required and contains basic information about the reso
   "group": "networking.istio.io",
   "version": "v1alpha3",
   "scope": "namespace",
-  "defaultPlaceholder": "- not set -"
+  "defaultPlaceholder": "- not set -",
+  "description": "See the {{[docs](https://github.com/kyma-project/busola)}} for more information."
```

Comment:
We should either add the description using translations or add a note to the docs that it can also work with it. The translations topic is confusing for our users anyway so let's give them some guidance